k8s/environments/sbox/cluster-01-overlay/kustomization.yaml

```diff
@@ -14,6 +14,10 @@ bases:
 
 patchesStrategicMerge:
 
+#secrets-csi-driver patch
+- ../../../release/admin/secrets-csi-driver/patches/sbox/cluster-00/secrets-csi-driver.yaml
```

Comment:
you don't need per cluster patches as they're the same thing
also if you did need per cluster patches you've applied the same thing twice so that's not right

Reply:
amended to single patch